exla/README.md

```diff
@@ -44,6 +44,19 @@ EXLA relies on the [XLA](https://github.com/elixir-nx/xla) package to provide th
     * Error message: "this file was generated by an older version of protoc which is incompatible with your Protocol Buffer headers".
     * If you have `protoc` installed on your machine, it may conflict with the `protoc` precompiled inside XLA. Uninstall, unlink, or remove `protoc` from your path to continue.
 
+### Usage with Nerves
+
+To get EXLA to work on your target Nerves device, you need to add `build_embedded: true` to your app's config.
+
+For example, if you're building an app called `:hello_nx`, you set:
+
+```elixir
+config :hello_nx, build_embedded: true
+```
+
+You also need to set your `XLA_ARCHIVE_URL` variable to the correct `aarch64` archive.
```

Comment:
Perhaps we can improve the behavior for automatically setting this somehow?
My question would be how to decide on the target architecture, though

Reply:
@polvalente what is the target we currently infer?

Reply:
Ahh, it's because we use the host target at build time, I see.

Reply:
We could add something like `XLA_FULL_TARGET=aarch64-linux-gnu-cpu` and then pass that in mix.exs?

Reply:
That sounds like a better alternative!
Either that or a variable for setting the architecture (since we already have one for cpu/cuda/etc)

Reply:
Sounds good! https://github.com/elixir-nx/xla/pull/38